css/modules/objects/post/_post.scss

```diff
@@ -421,7 +421,7 @@ $quarantine-VERTICAL_RHTYHM_LOOSE: 2rem;
   padding: 2rem;
   border: 0.3rem solid $COLOR_SECONDARY_DARK;
   font-family: monospace;
-  font-size: 1.4rem;
+  font-size: 1.8rem;
```

Comment:
Maybe we can keep it `1.4rem` in our smallest viewports and move it up to `1.8rem` in the `post_md.scss`?